r/Dockerfile

```diff
@@ -95,6 +95,9 @@ COPY CONDARC ./.condarc
 
 ENV PATH /usr/lib/rstudio-server/bin/:${PATH}
 
+ENV LD_LIBRARY_PATH=${CONDA_DIR}/lib
+ENV PROJ_LIB=${CONDA_DIR}/share/proj
```

Comment:
conda base env is not activated on RStudio for some reasons. Setting this in the R session worked. Let's see if we can make it persists by setting it here. We probably need the gdal data too.